charts/lnd/templates/statefulset.yaml

```diff
@@ -153,6 +158,14 @@ spec:
             mountPath: /root/.lnd
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          {{- if .Values.lndWatchOnly.enabled }}
+          command:
```

Comment:
We might not need a custom command to base64 decode the macaroon if we skip base64 encoding it before passing it to `kubectl create secret`.

Reply:
I found the same problem when exporting the `signer.custom.macaroon` to the secrets from the file. 
It did not decode to the same thing when decoded to be mounted in the pod so using the base64 encoded macaroon here.

This decode step is using the secret which was created from:
```
export MACAROON=$(base64 /root/.lnd/data/chain/bitcoin/{{.Values.global.network}}/signer.custom.macaroon | tr -d '\n\r')
 ```
I think this already solves the problem you mentioned and tested ok.